### ISSUE TITLE
Feature Request: control name of downloaded tempfile

### ISSUE THREAD

**Issue by softwaregravy** [janko/down#92]
Currently the name of the tempfile is hardcoded. I'd like the ability to change this when calling download. Sort of like setting the destination, but no need for the copy. I am interacting.

Thoughts on supporting a change like this?

https://github.com/janko/down/blob/c9737f9da7fe1424ae3057fb390f8ad1505e7a3e/lib/down/net_http.rb#L186C33-L186C46

**Comment by janko** [comment]
Yes, I would accept this change 👍🏻 Would you like to submit a PR?

**Comment by softwaregravy** [comment]
I'll try to get to this this week
